app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, file, directory = command.split()
+    if command == "mv":
+        if "/" in directory:
+            command_ls = directory.split("/")
+        else:
+            command_ls = directory
+        for direction in command_ls[:-1]:
+            file_directory = directory[: directory.index(direction)]
+            os.mkdir(f"{file_directory}{direction}")
```

Comment:
Is your code cross-platform?)For example, will it work with \ in command on windows?)

HINT: you can try to use os.path.split() to split directories and file_name in the path, os.makedirs() - to create directories without loop, and whatever you want)